server/src/models/Route.js

```diff
@@ -66,8 +84,35 @@ class Route extends Model {
    * Returns the full route after querying it, generally from the Popup
    * @param {number} id
    */
-  static async getOne(id) {
-    const result = await this.query().findById(id)
+  static async getOne(id, { isMad }) {
+    let result = null;
```

Comment:
Can we use a ternary here? React dev in me tries to avoid mutations when I can :)

Reply:
I'm struggling to let eslint autoformat my document, but I swapped to ternary as pr requested.